cms/djangoapps/contentstore/views/course.py

```diff
@@ -451,7 +451,7 @@ def course_filter(course_summary):
     courses_summary = get_courses_by_status(active_only, archived_only, courses_summary)
     courses_summary = get_courses_by_search_query(search_query, courses_summary)
     courses_summary = get_courses_order_by(order, courses_summary)
-    courses_summary = filter(course_filter, courses_summary)
+    courses_summary = list(filter(course_filter, courses_summary))
```

Comment:
Note to reviewer: I'm a bit worried about the performance impact of this change, but this was one of the alternatives which included fewer changes to the current source code.